tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/Handler.cs

```diff
@@ -161,17 +162,28 @@ private T DeserializePayload(string json)
             return payload;
         }
 
-        public async Task HandleAsync(string json, CancellationToken cancellationToken)
+        public async Task HandleAsync(IEnumerable<string> payloads, CancellationToken cancellationToken)
         {
-            var deserializedPayload = DeserializePayload(json);
-            var installationId = deserializedPayload.Installation.Id;
-
-            var client = await this.GitHubClientProvider.GetInstallationClientAsync(installationId, cancellationToken);
-            var context = new HandlerContext<T>(deserializedPayload, client);
-
-            await HandleCoreAsync(context, cancellationToken);
+            // TODO: Handle exceptions on deserialization (too big, malformed, shouldn't
```

Comment:
The various failure and retry cases seems like the biggest open question with this implementation. Goes directly to the retry logic in the event-hub and a per/multi message basis.